pkg/error/error.go

```diff
@@ -86,69 +87,123 @@ func ConsolidateErrors(errors []ValidationError, config config.Config) []Validat
 	return append(lineLessErrors, consolidatedErrors...)
 }
 
-// FormatErrors prints the errors to the console
-func FormatErrors(errors []ValidationErrors, config config.Config) []logger.LogMessage {
-	var formatted_errors []logger.LogMessage
-	var codeclimateIssues []CodeclimateIssue
+func FormatErrorsAsHumanReadable(errors []ValidationErrors, config config.Config) []logger.LogMessage {
+	var logmsgs []logger.LogMessage
 
 	for _, fileErrors := range errors {
-		if len(fileErrors.Errors) > 0 {
-			relativeFilePath, err := files.GetRelativePath(fileErrors.FilePath)
+		if len(fileErrors.Errors) == 0 {
+			continue
+		}
+
+		relativeFilePath, err := files.GetRelativePath(fileErrors.FilePath)
+		if err != nil {
+			logmsgs = append(logmsgs, logger.LogMessage{Level: "error", Message: err.Error()})
+			continue
+		}
+
+		fileErrors.Errors = ConsolidateErrors(fileErrors.Errors, config)
 
-			if err != nil {
-				formatted_errors = append(formatted_errors, logger.LogMessage{Level: "error", Message: err.Error()})
+		logmsgs = append(logmsgs, logger.LogMessage{Level: "warning", Message: fmt.Sprintf("%s:", relativeFilePath)})
+		for _, singleError := range fileErrors.Errors {
+			if singleError.LineNumber == -1 {
+				logmsgs = append(logmsgs, logger.LogMessage{Level: "error", Message: fmt.Sprintf("\t%s", singleError.Message)})
 				continue
 			}
 
-			// for these formats the errors need to be consolidated first.
-			if config.Format != "gcc" {
-				fileErrors.Errors = ConsolidateErrors(fileErrors.Errors, config)
+			if singleError.AdditionalIdenticalErrorCount == 0 {
+				logmsgs = append(logmsgs, logger.LogMessage{Level: "error", Message: fmt.Sprintf("\t%d: %s", singleError.LineNumber, singleError.Message)})
+				continue
+			}
+
+			logmsgs = append(logmsgs, logger.LogMessage{Level: "error", Message: fmt.Sprintf("\t%d-%d: %s", singleError.LineNumber, singleError.LineNumber+singleError.AdditionalIdenticalErrorCount, singleError.Message)})
+		}
+	}
+
+	return logmsgs
+}
+
+func FormatErrorsAsGHA(errors []ValidationErrors, config config.Config) []logger.LogMessage {
+	var logmsgs []logger.LogMessage
```

Comment:
```suggestion
	var logMessages []logger.LogMessage
```

or


```suggestion
	var logMsgs []logger.LogMessage
```

I prefer the first one

Reply:
done